src/openrct2/paint/tile_element/Paint.Surface.cpp

```diff
@@ -1015,7 +1015,7 @@ void surface_paint(paint_session* session, uint8_t direction, uint16_t height, c
         descriptor.corner_heights.left = baseHeight + ch.left;
     }
 
-    if ((session->ViewFlags & VIEWPORT_FLAG_LAND_HEIGHTS) && (zoomLevel == 0))
+    if (session->ViewFlags & VIEWPORT_FLAG_LAND_HEIGHTS)
```

Comment:
I don't think we should be enabling them on all zoom levels. You already mentioned they aren't really readable when zoomed out, anyway. Only showing them when zoomed **in** would be best, I think:

```suggestion
    if ((session->ViewFlags & VIEWPORT_FLAG_LAND_HEIGHTS) && (zoomLevel <= 0))
```

Reply:
I adjusted this and ensured that all the other height markers follow the exact same logic.